openapi-diff/src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -124,7 +124,10 @@ public override IEnumerable<ComparisonMessage> Compare(ComparisonContext context
             if (previousDefinition == null)
                 throw new ArgumentException("Comparing a service definition with something else.");
 
-            if (Info != null && previousDefinition.Info != null)
+            if (Info != null && 
+                previousDefinition.Info != null &&
+                Info.Version != null && 
+                previousDefinition.Info.Version != null)
```

Comment:
you can use sequential check to simpify : 

`Info?.Version != null && previousDefinition.Info?.Version != null`

Reply:
Done